server/groups.py

```diff
@@ -0,0 +1,15 @@
+import requests
+from flask import request
+
+
+def get_invites_for_account(account):
+    if account:
+        x_authorization = request.headers.get('X-Authorization')
+        authorization = request.headers.get('Authorization')
+
+        headers = {'Authorization': authorization if authorization else x_authorization}
+        invite_url = 'https://studentlife.pennlabs.org/users/{}/invites'.format(account.pennkey)
```

Comment:
Why do we need to provide the user's pennkey? The access token should be enough to determine who they are. Also, isn't it security issue if a user can provide their access token but use someone else's pennkey to get their invites/groups? (This is question about the gsr server, shouldn't hold up this request).